app/controllers/citations_controller.rb

```diff
@@ -4,8 +4,10 @@
 class CitationsController < ApplicationController
   before_action :authenticate
   before_action :load_info_request_and_authorise
+  before_action :set_in_pro_area
```

Comment:
I think this change was separate commit previously. Was it meant to be merged into this one?